mocket/mocket.py

```diff
@@ -615,5 +615,36 @@ def wrapper(t, *args, **kw):
 
         return decorator.decorator(wrapper, test)
 
+    async def __aenter__(self):
```

Comment:
I would move this block to a new file with an `if` for excluding it from Python `2.7` where the `async` keyword does not exist.

Reply:
Done but I think flake8 on python2 should ignore the new file, do you have any idea on how to achieve that?

Reply:
Have a look at this: https://flake8.pycqa.org/en/latest/user/options.html#cmdoption-flake8-per-file-ignores

Reply:
Yes, but this has to be done on a per version basis.

Reply:
Maybe travis gives some env var for the version of python, or it has to be done using bash code

Reply:
> Yes, but this has to be done on a per version basis.

Not really, we just have to ignore `E999 SyntaxError` for that specific file.

Reply:
But that would be bad for versions that support async.
I know the error would be cached afterwards, but it should be flake8's job to catch it.

Reply:
That's why I asked you to move the code to that specific file. So `flake8` will still check the rest of the project. Keep in mind that at some point in the future I will deprecate Python `2.7`, so it's a temporary fix.

Reply:
Of course feel free to investigate another approach. It's your PR and I'll leave it to you. Thank you for your contribution, btw.